problems/getCountriesSortedByPopulation.js

```diff
@@ -10,8 +10,14 @@
 * 
 */
 
-function getCountriesSortedByPopulation() {
-
+function getCountriesSortedByPopulation(arr) {
+    let sortedArr = arr.sort((a,b) => {
+        return a.population - b.population
```

Comment:
Consider what b.population - a.population will return. Do you need a .reverse() on line 20?